### ISSUE TITLE
Uncaught Error: Requested measurement of a line component that is not currently rendered

### ISSUE THREAD

**Issue by danwhitston** [atom/atom#17294]
1. Error did not repeat after file was saved, closed and reopened.
2. Searching for a simple string in a Ruby file with unsaved modifications (searching on `should `, manually replacing highlighted words).
3. Search reached a section of indented code that was hidden using those little arrows on the left.
4. The below error was thrown, popping up continually until the file was saved and the pane shut.

I'm not that bothered as it's not recurring currently, but thought you might appreciate the report.

**Atom**: 1.26.1 x64
**Electron**: 1.7.11
**OS**: Microsoft Windows 10 Pro
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Requested measurement of a line component that is not currently rendered

```
At C:\Users\Daniel\AppData\Local\atom\app-1.26.1\resources\app\src\text-editor-component.js:2287

Error: Requested measurement of a line component that is not currently rendered
    at horizontalPositionsToMeasure.forEach (~/AppData/Local/atom/app-1.26.1/resources/app/src/text-editor-component.js:2276:29)
    at Map.forEach (native)
    at TextEditorComponent.measureHorizontalPositions (~/AppData/Local/atom/app-1.26.1/resources/app/src/text-editor-component.js:2269:45)
    at TextEditorComponent.measureContentDuringUpdateSync (~/AppData/Local/atom/app-1.26.1/resources/app/src/text-editor-component.js:409:16)
    at scheduler.readDocument (~/AppData/Local/atom/app-1.26.1/resources/app/src/text-editor-component.js:279:41)
    at ViewRegistry.performDocumentUpdate (~/AppData/Local/atom/app-1.26.1/resources/app/src/view-registry.js:245:13)
```

### Commands

```
     -3:07.4.0 core:move-right (input.hidden-input)
  5x -3:03.1.0 core:delete (input.hidden-input)
     -2:29 editor:move-to-end-of-word (input.hidden-input)
     -2:28.6.0 core:move-left (input.hidden-input)
  3x -2:28.4.0 core:move-right (input.hidden-input)
     -2:15.6.0 core:delete (input.hidden-input)
     -2:14.6.0 editor:move-to-end-of-word (input.hidden-input)
  2x -2:12.9.0 core:move-right (input.hidden-input)
     -2:07.1.0 core:save (input.hidden-input)
     -1:57.8.0 find-and-replace:show (input.hidden-input)
  3x -1:56 core:confirm (input.hidden-input)
     -1:47.9.0 editor:delete-line (input.hidden-input)
     -1:41.2.0 find-and-replace:show (input.hidden-input)
 30x -1:39.6.0 core:confirm (input.hidden-input)
     -0:21.4.0 editor:consolidate-selections (input.hidden-input)
     -0:21.4.0 core:cancel (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.32.2 
busy-signal 1.4.3 
intentions 1.1.5 
language-haml 0.25.2 
language-liquid 0.6.0 
line-count 0.5.0 
linter 2.2.0 
linter-reek 2.2.2 
linter-rubocop 2.2.3 
linter-ui-default 1.7.1 
minimap 4.29.8 
tag 0.5.0 
```

**Comment by rsese** [comment]
Thanks for the report!

Based on the stack trace, looks like a duplicate of https://github.com/atom/atom/issues/15263 which I think we were hoping would be fixed in 1.26 . I'll go ahead and re-open https://github.com/atom/atom/issues/15263 and copy your steps there.

**Comment by rsese** [comment]
Re-opening per https://github.com/atom/atom/issues/15263#issuecomment-388180825.    If you come across reproducible steps @danwhitston, please let us know, we do appreciate the report :v:
